src/website/app/Markdown.js

```diff
@@ -211,6 +199,29 @@ export default function Markdown({ children, scope, ...restProps }: Props) {
       },
       table({ children }) {
         return <MarkdownTable>{children}</MarkdownTable>;
+      },
+      li({ children }) {
+        const newChildren = children.map(child => {
+          if (!(child && child.split)) {
+            return child;
+          }
+
+          const [preLabelText, labelText] = child.split(/ *~ */);
+
+          if (!labelText) {
+            return child;
+          }
+
+          const labelVariant = 'done' === labelText ? 'success' : 'regular';
+
+          return [
+            preLabelText,
+            <Label key={1} variant={labelVariant}>
```

Comment:
The static `className` I suggested in the preceding comment could be added here, to avoid placing it in the Label component directly.

Reply:
fixed